sncast_scripts/src/declare_presets/declare_presets.cairo

```diff
@@ -0,0 +1,47 @@
+use openzeppelin_testing::common::IntoBase16String;
+use sncast_std::{declare, get_nonce, FeeSettings, EthFeeSettings};
+
+const MAX_FEE: felt252 = 99999999999999999;
+
+fn main() {
+    let contracts: Array<ByteArray> = array![
+        "AccountUpgradeable",
+        "ERC20Upgradeable",
+        "ERC721Upgradeable",
+        "ERC1155Upgradeable",
+        "EthAccountUpgradeable",
+        "VestingWallet"
+    ];
+
+    let mut consume_latest_nonce = false;
+    let mut nonce = get_nonce('latest');
+
+    for contract in contracts {
+        if (!consume_latest_nonce) {
+            consume_latest_nonce = true;
+        } else {
+            nonce = get_nonce('pending');
+        }
+
+        declare_preset(contract, nonce);
+    };
+
+    println!("");
+}
+
+fn declare_preset(contract: ByteArray, nonce: felt252) {
+    println!("\nDeclaring {contract}:");
+
+    let declare_result = declare(
+        contract,
+        FeeSettings::Eth(EthFeeSettings { max_fee: Option::Some(MAX_FEE) }),
+        Option::Some(nonce)
+    );
+
+    // Print output
+    // Note that the tx hash will already display from the declaration
+    match declare_result {
+        Result::Ok(r) => println!("Class hash = {}", r.class_hash.into_base_16_string()),
+        Result::Err(r) => println!("{:?}", r)
+    }
```

Comment:
Will it result in an error if class is already declared?

Reply:
It will be an error but the execution won't stop since this proposes to print errors (in case erc20 is declared but the rest are not, for example). Right now, an already declared class returns the generic `REJECTED` instead of `ClassAlreadyDeclared`. Considering this and the "Experimental" disclaimer on SN Foundry's [script page](https://foundry-rs.github.io/starknet-foundry/starknet/script.html), I'm proposing to just print errors until this feature is improved. The script is just for our team after all